libraries/AP_Motors/examples/AP_Motors_test/AP_Motors_test.cpp

```diff
@@ -74,7 +74,7 @@ void setup()
     // set rc channel ranges
     rc1.set_angle(4500);
     rc2.set_angle(4500);
-    rc3.set_range(130, 1000);
+    rc3.set_range(1000);
```

Comment:
this is a functional change to the bottom-end of the throttle range. is it important?